rero_ils/modules/locations/jsonschemas/locations/location-v0.0.1.json

```diff
@@ -112,6 +117,84 @@
           "pattern": "^https://ils.rero.ch/api/libraries/.*?$"
         }
       }
+    },
+    "allow_request": {
+      "title": "Allow request",
+      "type": "boolean",
+      "description": "If enabled, it allows requests for items linked to this location.",
+      "default": true
+    },
+    "send_notification": {
+      "title": "Send notification",
```

Comment:
I would change to: Email notification, but perhaps this has been validated by the PO.